lib/benchee/formatters/html.ex

```diff
@@ -114,51 +115,53 @@ defmodule Benchee.Formatters.HTML do
 
   Returns a map from file name/path to file content.
   """
-  @spec format(Benchee.Suite.t) :: %{Benchee.Suite.key => String.t}
-  def format(%{statistics: statistics, run_times: run_times, system: system,
+  @spec format(Suite.t) :: %{Suite.key => String.t}
+  def format(%Suite{scenarios: scenarios, system: system,
                configuration: %{
                  formatter_options: %{html: %{file: filename}}}}) do
-    statistics
-    |> input_job_reports(run_times, system, filename)
+    scenarios
+    |> Enum.group_by(fn(scenario) -> scenario.input_name end)
+    |> Enum.map(fn(tuple) -> reports_for_input(tuple, system, filename) end)
```

Comment:
Or maybe `&1` :trollface: 

`tagged_scenarios` is a much better name - I'll go with that!